os/selinux.md

```diff
@@ -27,11 +27,11 @@ Once you're happy that your workload is compatible with the SELinux policy,
 you can temporarily enable enforcement by running the following command as
 root:
 
-* setenforce 1
+* `setenforce 1`
 
 To enable it across reboots, do the following:
 
-* `cp --remove-destination $(readlink /etc/selinux/config) /etc/selinux/config`
+* `cp --remove-destination $(realpath /etc/selinux/config) /etc/selinux/config`
```

Comment:
readlink returns actual symlink (../../usr/path/to). realpath returns absolute path to symlink (/usr/path/to)